File: PythonSolutions/level_two/rotate_array.py
def rotate_array(arr):
    last_index = len(arr) - 1
    for first_index in range(last_index):
        for second_index in range(len(arr)):
            temp = arr[first_index]
            arr[first_index] = arr[last_index]
            arr[last_index] = temp
    print(arr)


if __name__ =='__main__':
    numbers = [1, 2, 3, 4, 5]
    rotate_array(numbers)

File: JavaSolutions/levelTwo/RotateArray.java
package semicolonClassWork.levelTwo;

import java.util.Arrays;

public class RotateArray {
    public static void main(String[] args) {
        int[] numbers = {1, 2, 3, 4, 5};
        rotateArray(numbers);
    }

    private static void rotateArray(int[] arr){
        int lastIndex = arr.length-1;
        for (int firstIndex = 0; firstIndex < lastIndex; firstIndex++){
            for (int secondIndex = 0; secondIndex < arr.length; secondIndex++){
                  int temp = arr[firstIndex];
                  arr[firstIndex] = arr[lastIndex];
                  arr[lastIndex] = temp;
            }
        }
        System.out.println(Arrays.toString(arr));
    }
}
